apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -40,7 +42,34 @@ const ReviewWork = ({ issue, submissionIndex, readOnly }) => {
 
   const [ feedback, setFeedback ] = useState('')
   const [ rating, setRating ] = useState(-1)
-  const [ index, setIndex ] = useState(submissionIndex)
+
+  const submissionsApproved = issue.workSubmissions.reduce((counter, submission) => {
+    if ('review' in submission && submission.review.accepted) counter++
+    return counter
+  }, 0)
+
+  const canReview = submissionsApproved < (issue.bounties ? issue.bounties : 1) && !readOnly
```

Comment:
So you can only review if there are fewer reviewed submissions than the total number of bounties for this issue?

Two questions:

* do we actually support `issue.bounties` yet? Is this an interface we are planning to add, but have not yet implemented?
* is there a possibility that there would be N bounties and more than N submissions? If so, why do we want to forbid reviewing submission N+1? I think the fact that I'm even asking this question right now _probably_ means that we're being unnecessarily complex with this statement – there's probably a simpler way to write logic about what our system actually does.

Reply:
> So you can only review if there are fewer reviewed submissions than the total number of bounties for this issue?

Yes. Fewer approved submissions.

> do we actually support issue.bounties yet? 

Not just yet, but hopefully soon.

> is there a possibility that there would be N bounties and more than N submissions

Yes. But only N can be approved, hence the logic... slightly improved now though :)